### ISSUE TITLE
Unify paths and urls into less options for future scalable and easy usability.

### ISSUE THREAD

**Issue by Artanias** [OSLL/code-plagiarism#148]
For the easy of using utility need to try merge two arguments into one and correctly define which url gotten in the logic automaticaly:
`codeplag check --github-files` and `codeplag check --github-project-folders` -> `codeplag check --github-url`

**Comment by PeeachPie** [comment]
Мы хотим url до папки? Новый функционал должен соответствовать `-gf url/file_1 ... url/file_N -gp url`? Или мы хотим url до файла и `-gf url -gp url../` (или -gp корень репо)

**Comment by Artanias** [comment]
@PeeachPie, сейчас `--github-project-folders` натравливается на конкретную папку, например, `https://github.com/OSLL/code-plagiarism/blob/main/src/codeplag/pyplag`.
`--github-files` натравливается на конкретный файл, например, https://github.com/OSLL/code-plagiarism/blob/main/src/codeplag/pyplag/utils.py.

Соответственное функционал этих двух флагов теперь должен быть спрятам под одним флагом `--github-url`, а в логике уже определять что это, папка или файл. По ответу с `GitHub` можно даже это сделать.

**Comment by Artanias** [comment]
@PeeachPie, нет, старые можно будет удалить, изменение пойдёт в `0.6.x` версию, поэтому обратная совместимость не нужна будет.

**Comment by PeeachPie** [comment]
> Теперь?

Нет

> В данном случае ПО натравливается на Ваши репозитории и в зависимости от токена, либо только публичные, либо также и приватные

Теперь понял для чего он)

**Comment by Artanias** [comment]
> Нет

Теперь точно должен быть.
